src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -6057,8 +6057,8 @@ static void Main()
             var x2Ref = GetReference(tree, "x2");
             Assert.Equal("string", model.GetTypeInfo(x2Ref).Type.ToDisplayString());
 
-            VerifyModelForDeconstructionLocal(model, x1, x1Ref);
-            VerifyModelForDeconstructionLocal(model, x2, x2Ref);
+            VerifyModelForDeconstruction(model, x1, LocalDeclarationKind.OutVariable, x1Ref);
+            VerifyModelForDeconstruction(model, x2, LocalDeclarationKind.OutVariable, x2Ref);
```

Comment:
That is what results from the way they are handled in the expression variable finder. It is arbitrary because this is an error, but this is the simplest implementation.